// check_error 3:21 "128 is out of range for type 'int8'"
func main() {
    const b: int8 = 128;
}
